Classes/Domain/Model/Book.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types=1);
+
+namespace TTN\Books\Domain\Model;
+
+use TYPO3\CMS\Extbase\DomainObject\AbstractEntity;
+
+/**
+ * Domain model for a book.
+ */
+final class Book extends AbstractEntity
```

Comment:
Please let's not have this final. This is a teaching example, and the model classes of real-world extensions should be extendable via XClasses.